### ISSUE TITLE
Not syncing two Libraries at the same time ( Mobile and Desktop )

### ISSUE THREAD

**Issue by MuriloNG** [romannurik/Sketch-Stickers#10]
Hello Romannurik,

I really appreciate your plugin, it is awesome.
Right now in the documentation it says it is possible to sync it with one or more libraries.

I just tried to build one library for desktop and a second one for Mobile.
In the code page, the names of the components are different for mobile and desktop.
But it only syncs one of them per time. If I link the mobile library I can not use the desktop one and if I deactivate the mobile library I can use the desktop library again.

![desktop library](https://user-images.githubusercontent.com/41154026/43394911-9258e168-93fc-11e8-83bd-f034c367e1a0.png)
![mobile library](https://user-images.githubusercontent.com/41154026/43394912-929cdd82-93fc-11e8-949d-241d8364c8a6.png)

Is there anything we could add in the code to make it work for both at the same time?

Thanks in advance.
Best,
Murilo

**Comment by romannurik** [comment]
Hi Murilo, this could be related to libraries sharing an ID, try looking at issue #9 to see if this is related? https://github.com/romannurik/Sketch-Stickers/issues/9#issuecomment-406145113 -- there's a code snippet you can run in Plugins > Run script that'll tell you of libraries are sharing an ID

**Comment by MuriloNG** [comment]
1. In "Sketch/Plugins/Run Plugin Script"  I pasted the script.

let libsById = {};
let libraries = Array.from(NSApp.delegate().librariesController().libraries())
  .filter(lib => !!lib.locationOnDisk() && !!lib.enabled() && !!lib.libraryID())
  .forEach(lib => {
    let id = String(lib.libraryID());
    libsById[id] = libsById[id] || [];
    libsById[id].push(lib);
  });
let dups = Object.values(libsById).filter(libs => libs.length > 1);
if (!dups.length) {
  log('All good! None of your libraries share an ID');
  return;
}

dups.forEach(libs => {
  log('These libraries share an ID:');
  libs.forEach(lib => log('> ' + lib.name()));
  log('');
});


And you are right. They have the same ID. That was the issue.


2. One possible solution could be to use the Sketch Automate Plugin. 
"Sketch/Plugins/Automate/Libraries/Fix Library ID conflict" and change the Library ID.

I think it could work, but we didn't tested it. We decide to merge the 2 libraries "Mobile" and "Desktop" in one Sketch Library file.

I just wrote it here detailed if someone in the future needs help about the same topic.
Thanks so far.

Best,
Murilo
